### PR TITLE
Enemy movement

### PR DESCRIPTION
**Pull request by matharotheelf**

### CODE REVIEW

**Review by hotblack86 — APPROVED** [review]
Good work. Very neat.

**Review by hotblack86 — APPROVED** [review]

**Review by CKKH — APPROVED** [review]
Nice work all - skeleton pngs incoming

File: src/main.py [hunk 1]
```diff
@@ -20,29 +20,38 @@ def __init__(self):
...
         self.all_sprites = pg.sprite.Group()
+        self.enemy_sprites = pg.sprite.Group()
         self.hero = None
+        self.enemy = None
 
         self.screen = pg.display.set_mode((WIDTH, HEIGHT))
         self.playing = None
         self.map_data = []
         self.load_data()
+        self.counter = 0
```
Comment (michaelnguyen974) [inline src/main.py:31]: Nice use of the counter instead of the clock

File: src/main.py [hunk 2]
```diff
@@ -20,29 +20,38 @@ def __init__(self):
...
+            self.counter += 1
 
     def new(self):
         ''' Creates sprites '''
 
-        self.all_sprites.add(Enemy(self, 1, 1))
+        self.all_sprites = pg.sprite.Group()
+        self.enemy = Enemy(self, 1, 1)
         self.hero = Hero(self, 5, 5)
+        self.walls_sprites = pg.sprite.Group()
```
Comment (michaelnguyen974) [inline src/main.py:50]: Good separation of sprites, this will allow us to create more enemies without any bugs creeping up on us.

**Review by michaelnguyen974 — APPROVED** [review]

**Pull request merged**